strings/tips/xeno.txt

```diff
@@ -1,59 +1,56 @@
+Before you evolve into the Hivemind as a drone, hide in a secure area! Once you evolve, you will create a core that cannot be moved.
 As a carrier, you can hit a host directly with a thrown facehugger to have a high chance of infecting them near instantly. This will take time, however.
 As a carrier, you can hide traps beneath certain objects and items.
 As a hunter, you have almost perfect stealth if you stand still.
-As a hunter, you can sneak / change move intention during stealth to make it hard on marines to see you while moving in stealth.
-As a ravager or the Queen, you are fire-resistant, NOT fireproof. You can still get lit on fire, but you take very little damage from it.
-As a spitter, boiler, praetorian, or the Queen take advantage of unguarded barricades and defenses, melt or constantly spit acid on them if you can.
-As a boiler or the Queen, you have the most powerful corrosive acid for melting things.
-As the Shrike, take advantage of your psychic abilities. Use Psychic Fling and Unrelenting Force to displace a marine or even push certain and hazardous objects to your enemies and vice versa.
+As a ravager, the King or the Queen, you are fire-resistant, NOT fireproof. You can still get lit on fire, but you take very little damage from it.
+As a spitter, boiler, praetorian, or the Queen, take advantage of unguarded barricades and defenses; melt or constantly spit acid on them if you can.
+As a boiler, the King or the Queen, you have the most powerful corrosive acid for melting things.
+As the Shrike, take advantage of your psychic abilities. Use Psychic Fling and Unrelenting Force to displace a marine or even push certain and hazardous objects towards your enemies and vice versa.
 As the Queen, your screech does not only affect the mobs on view, but affects through walls or gases. However, the effect of the screech will be reduced.
-As a drone, hivelord, hivemind, shrike, and queen, a large maze is almost always a good investment.
-As a drone, hivelord, hivemind, shrike, and queen, be clever with your walling. Try to allow for as much fluid movement as possible.
+As a drone, hivelord, Hivemind, shrike, and the Queen, a large maze is almost always a good investment.
+As a drone, hivelord, Hivemind, shrike, and the Queen, be clever with your walling. Try to allow for as much fluid movement as possible.
 Sticky resin can act as the path for your maze to be built around.
 The Hivemind is very skilled at flanking marines. Wait for them to push, and wall them off from behind!
 Praying to the Queen-Mother will usually net a response.
 Your queen is your lifeline. Try to listen to her.
 As a hunter, the movement of your Mirage is controlled by your intent.
 Defenders have a very small window to sweep grenades back at marines with their tail.
 Aiming for the head is a good long-term investment, causing brain damage on marines.
-Aiming for the groin can easily damage someones liver.
```

Comment:
this is still true

Reply:
liver is a part of the chest, do you want it just changed to "aiming for the chest"?

/code/modules/organs/organ_internal.dm
```/datum/internal_organ/liver
	name = "liver"
	parent_limb = "chest"```

Reply:
yep. Probably expand it to the other organs there.